src/index.ts

```diff
@@ -134,17 +140,26 @@ export class I2C extends Peripheral {
     super.destroy();
   }
 
-  private getDevice(address: number) {
+  private getDevice(address: number, busNumber?: number) {
     let device = this.devices[address];
 
     if (device === undefined) {
-      device = openSync(getBoardRevision() === VERSION_1_MODEL_B_REV_1 ? 0 : 1);
+      if (busNumber === undefined) {
+        busNumber = this.busNumber;
+      }
+      device = openSync(busNumber);
       this.devices[address] = device;
     }
 
     return device;
   }
 
+  public configure(options?: any): void {
```

Comment:
> This is curious though, with the APIs defined the way they are for IO plugins, it's not possible to talk to two devices with the same address but each on a different bus.

There's an issue for that [here](https://github.com/rwaldron/io-plugins/issues/13).